### ISSUE TITLE
Sound bug

### ISSUE THREAD

**Issue by Dert269** [Zet0rz/nzombies#460]
Every time I load up a map and edit it in Creative mode, every time a prop scrapes against something in the map, it causes this annoying loud sound, that transfers to Survival mode, this bug doesn't happen when I'm playing on maps that I didn't make, and it only occurs when I go into Creative mode.

**Comment by DjNathann** [comment]
Does it do the sound of "Bang"?

**Comment by Dert269** [comment]
No, it makes this horrible screeching noise and it happens with all props on the map, even ejected bullet shells, when I get a moment today, I'll record it and provide you with a link.

**Comment by DjNathann** [comment]
Zet0rz wont be here in a couple of weeks.

so you gotta wait...

**Comment by Dert269** [comment]
Alright, but I'll still get the video ready.

**Comment by Dert269** [comment]
Nevermind, I found the problem, I had to redownload the latest version of nZombies from this website, rather than the Steam Workshop version.
